pkg/framework/context/context.go

```diff
@@ -91,10 +82,10 @@ func WithOutboundTransport(ot transport.OutboundTransport) ProviderOption {
 	}
 }
 
-// WithProtocols injects protocol svc into context
-func WithProtocols(protocolSvcCreator ...api.ProtocolSvcCreator) ProviderOption {
+// WithProtocolServices injects protocol services into the context.
```

Comment:
Why did we need to rename this method?

Reply:
Before this change Context required `Protocol` to be passed in to create the `ProtocolServices`. Now services are created in framework.New(), we can directly pass `ProtocolServices` to context.